source/scss/moon.scss

```diff
@@ -0,0 +1,21 @@
+/*
+ * Moon
+ *
+ */
+@import 'shared';
+
+%moon,
+.moon {
+  @extend %filter-base;
+  filter: grayscale(1) contrast(1.1) brightness(1.1);
+
+  &:before {
+    background: rgb(180, 178, 178);
```

Comment:
Try: `rgb(160, 160, 160)` to make the shadows a bit darker. Otherwise, this PR looks :sparkles:*awesome*:sparkles: